templates/standby-clone.sh.j2

```diff
@@ -29,11 +29,8 @@ sudo -u postgres mkdir -p /var/lib/postgresql/{{ postgres_version }}/{{ postgres
 
 echo Get previous backup from backups server
 sudo -u postgres \
-  time rsync --progress -pvia --exclude='*.conf' --exclude='server.crt' --exclude='server.key' --delete \
-{% if postgres_barman_rsync_enabled|default(false) -%}
-  --password-file=/var/lib/postgresql/.rsync_pass \
```

Comment:
what happens to this default value that disappears from your change?

Reply:
I have re-added it in the `set_fact` step (if using rsync)